### ISSUE TITLE
Unable to open OR App

### ISSUE THREAD

**Issue by drasticdpk** [topcoder-platform/work-manager#1069]
I have created the task from TCx  app(https://x.topcoder.com/). And it shows the Payment was done but I am unable to open that challenge from work manager. 

![image](https://user-images.githubusercontent.com/25297859/108164739-7e2e0b80-7119-11eb-8a9e-4c06c4ccac4c.png)

**Reference** 

![image](https://user-images.githubusercontent.com/25297859/108164589-2f807180-7119-11eb-8773-6bf79112d4be.png)
![image](https://user-images.githubusercontent.com/25297859/108164652-4b841300-7119-11eb-8861-9ddff6dd6423.png)

https://www.topcoder.com/challenges/f6513dac-52b4-4d6e-917b-a458f38e8aa4
https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30168461

**Comment by maxceem** [comment]
@vikasrohit I guess this is something to do with permissions and legacy syncing, so I'm not sure if we can include this into `3/12` release.

**Comment by vikasrohit** [comment]
Yes, it was an issue this week which @rootelement already took care of. @drasticdpk can you please confirm, if you are still seeing the issue?

**Comment by drasticdpk** [comment]
Hi 
Still having same issue for me. 
Temp Solution :  For now I am creating another task and paid the developer. Once the issue is fixed I will delete that task from OR.

**Comment by vikasrohit** [comment]
ok, it might be happening for the existing challenge, we might need to fix the data for that. But you should not be seeing the issue for your new challenges.
